pkgs/development/python-modules/flask-paranoid/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, flask
+}:
+
+buildPythonPackage rec {
+  pname = "flask-paranoid";
+  version = "0.2.0";
+
+  src = fetchPypi {
+    pname = "Flask-Paranoid";
+    inherit version;
+    sha256 = "ou6SV2QuAlTxFXikqlShpyUWemQc+gh5Mw1HfsNwx0k=";
+  };
+
+  propagatedBuildInputs = [
+    flask
+  ];
+
+  pythonImportsCheck = [ "flask_paranoid" ];
+
+  # PyPi version does not include test directory
+  doCheck = false;
```

Comment:
Can we fetch from GitHub and execute tests? They tag releases https://github.com/miguelgrinberg/flask-paranoid/tree/v0.2.